src/core/server/core_usage_data/core_usage_data_service.ts

```diff
@@ -56,6 +55,8 @@ export interface StartDeps {
   exposedConfigsToUsage: ExposedConfigsToUsage;
 }
 
+const kibanaIndex = '.kibana';
```

Comment:
the `savedObjects` service depends on `coreUsageData`, and not the other way around, so we can't easily use the new `getKibanaIndex` API from within the `coreUsageData` service. Had to hardcode the value here, unfortunately.

Could eventually expose this constant from `src/core/saved_objects` and import it there, it would probably be slightly better, but OTOH this will likely not change, and this code is only internal to core, so it would be not impact API consumers if we need to refactor it in the future.